app/src/main/java/org/shadowice/flocke/andotp/Database/Entry.java

```diff
@@ -125,6 +125,9 @@ public Entry(String contents) throws Exception {
             case "steam":
                 type = OTPType.STEAM;
                 break;
+            case "pin":
```

Comment:
This function is for adding a new entry from an URI, if we add PIN functionality here we also need to add it to the `toUri` function below where you added the comment `qr code not supported`.
So we either remove it from both methods or add it to both.